_data/KiranAdhikari/KiranAdhikari.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kiran Adhikari
+institution: Tribhuvan University
+profile_pic: KiranAdhikari.jpg
+quote: "Premature optimization is the root of all evil"
```

Comment:
Remove double quotes from begin and end.

Reply:
Done